src/utils/poster-manager.js

```diff
@@ -76,6 +76,17 @@ class PosterManager {
   }
 
   /**
+   * set src & Show the poster image
+   * @public
+   * @param {string} posterUrl - the poster image URL
+   * @returns {void}
+   */
+  setAndShow(posterUrl: ?string): void {
```

Comment:
First, this method is redundant (see my comment above)
Second, the name of it is too generic - set and show what?